packages/next/client/future/image.tsx

```diff
@@ -35,6 +35,17 @@ type LoadingValue = typeof VALID_LOADING_VALUES[number]
 type ImageConfig = ImageConfigComplete & { allSizes: number[] }
 export type ImageLoader = (p: ImageLoaderProps) => string
 
+let globalImageConfig: Partial<ImageConfigComplete> = {}
+
+export function setImageConfig(imageConfig: Partial<ImageConfigComplete>) {
```

Comment:
Is it possible that the globalImageConfig changes during the lifecycle of app, or will only run once when the app start?

When a `next/future/image` is contained in a react subtree that has opted in concurrent features, changing the globalImageConfig might cause inconsistency between `next/future/image` components.